### PR TITLE
tools: Playbook to configure incerta back NIC

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: tools/incerta-nic.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      register: wrote_network_config
+      with_items:
+        - { regexp: '^DEVICE=', line: 'DEVICE={{ nic_to_configure }}' }
+        - { regexp: '^NAME=', line: 'NAME={{ nic_to_configure }}' }
+        - { regexp: '^BOOTPROTO=', line: 'BOOTPROTO=static' }
+        - { regexp: '^ONBOOT=', line: 'ONBOOT=yes' }
+        - { regexp: '^MTU=', line: 'MTU=9216' }
+        - { regexp: '^IPADDR=', line: 'IPADDR={{ incerta_back_ip }}' }
+        - { regexp: '^PREFIX=', line: 'PREFIX=24' }
+        - { regexp: '^DEFROUTE=', line: 'DEFROUTE=no' }
```
Comment (zmc) [inline tools/incerta-nic.yml:100]: Not sure if I've seen `lineinfile` used with `with_items` like this before. Nice!

File: tools/incerta-nic.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    
+    - name: "Bounce {{ nic_to_configure }}"
+      shell: "ifdown {{ nic_to_configure }}; ifup {{ nic_to_configure }}"
+      when:
+        - wrote_network_config is changed
+        - ansible_os_family == 'RedHat'
+    
+    - fail:
+        msg: "WARNING: {{ ansible_hostname }} IS USING NETPLAN TO CONFIGURE ITS NICS. EDITING NETPLAN YAML FILES USING ANSIBLE IS NOT TRIVIAL. THEREFORE, THIS NETWORK SETTING WILL NOT SURVIVE A REBOOT! RECOMMEND MANUALLY EDITING /etc/netplan/01-netcfg.yaml"
+      ignore_errors: true
```
Comment (zmc) [inline tools/incerta-nic.yml:113]: I'm curious - why `ignore_errors` with `fail:` ?

**Review by zmc — APPROVED** [review]
Looks good!

File: tools/incerta-nic.yml (hunk 2 above)
Comment (djgalloway) [reply tools/incerta-nic.yml:113]: Oh I guess I could get rid of it since it's the last task.

**Pull request merged**